pyemma/coordinates/api.py

```diff
@@ -288,7 +289,7 @@ def source(inp, features=None, top=None, chunk_size=100):
         # check: if single array, create a one-element list
         # check: do all arrays have compatible dimensions (*, N)? If not: raise ValueError.
         # create MemoryReader
-        reader = _DataInMemory(inp)
+        reader = _DataInMemory(inp, chunksize=chunk_size)
```

Comment:
The default chunksize for DataInMemory is 5000, because 100 was too small for most cases (high python overhead). But actually you're right to pass the argument there, because otherwise the user has to make an extra call to set the chunksize.